docs/dev/topology-over-raft.md

```diff
@@ -4,7 +4,8 @@ The topology state machine tracks all the nodes in a cluster,
 their state, properties (topology, tokens, etc) and requested actions.
 
 Node state can be one of those:
-- `none`             - the new node joined group0 but did not bootstrapped yet (has no tokens and data to serve)
+
+- `none`             - the new node joined group0 but has not bootstrapped yet (has no tokens and data to serve)
```

Comment:
"did not bootstrapped" --> should be either "did not bootstrap" or "has not bootstrapped yet" (and with the usage of "yet" the "has not bootstrapped" seems to be more grammatically correct)